src/main/Video.tsx

```diff
@@ -247,15 +247,21 @@ const VideoHeader: React.FC<{}> = () => {
   const presenters = useSelector(selectPresenters)
 
   const titleStyle = {
-    fontSize: 'large'
-  }
+    display: 'inline-block',
+    padding: '15px',
+    overflow: 'hidden',
+    whiteSpace: "nowrap",
+    textOverflow: 'ellipsis',
+    maxWidth: '500px'
+  } as React.CSSProperties;
+
 
   return (
-    <div title="Video Area Header">
-      <div css={titleStyle} title="Video Title">{title}</div>
-      <div title="Video Presenters">by {presenters.join(", ")}</div>
+    <div title="Video Area Header" css={{fontSize: '16px'}}>
+      <div style={titleStyle} css={{fontSize: '24px', fontWeight: 'bold'}} title="Video Title">{title}</div>
+      <div style={titleStyle} title="Video Presenters">by {presenters.join(", ")}</div>
```

Comment:
Instead of using the `style` tag (and splitting the css between the `style` and the `css` tag), I think it would be better to create multiple css objects and apply them using composition:
```suggestion
  const titleStyle = css({
    display: 'inline-block',
    padding: '15px',
    overflow: 'hidden',
    whiteSpace: "nowrap",
    textOverflow: 'ellipsis',
    maxWidth: '500px'
  })

  const titleStyleBold = css({
    fontWeight: 'bold',
    fontSize: '24px',
  })


  return (
    <div title="Video Area Header" css={{fontSize: '16px'}}>
      <div css={[titleStyle, titleStyleBold]} title="Video Title">{title}</div>
      <div css={titleStyle} title="Video Presenters">by {presenters.join(", ")}</div>
    </div>
  );
```
(Only the first three lines are marked as deleted because Github won't let me mark everything in one go)

Reply:
Are you certain which font-size will with if we do it like this?

Reply:
I am struggling to find concrete proof as to why exactly a childs CSS will always take precedence over a parents CSS, but I don't know why it should not be so.

Reply:
You are right. I misread the code. Sorry.

Reply:
Do you agree with suggested changes otherwise? Asking in the interest of getting this merged.

Reply:
Adopted your suggestions